### PR TITLE
Update ug dg rahul

### LINKED ISSUE #57: Update DG - Rahul

### PR DESCRIPTION
**Pull request by Rahul0506**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -112,45 +102,46 @@ Will eventually be extended to get help about specific commands.
...
 
 
 ### Adding a item: `add`
-NOT Supported as of v1.2 yet
+NOT Supported as of v1.2
 
 **NAME:**
 
-- `add` - add a quantity to a single specified item
+- `add` - adds quantity to a single specified item
```
Comment (aqcd) [inline docs/UserGuide.md]: is now supported as addq, should update this?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -112,45 +102,46 @@ Will eventually be extended to get help about specific commands.
...
+- Adds <u>8</u> more <u>Bob’s 6th regrets</u> to the inventory
 
 ### Listing all items or recipes : `list`
-NOT Supported as of v1.2 yet
+NOT Supported as of v1.2
 
 **NAME:**
 	
-- `list` - lists all items or recipes the user has entered
+- `list` - lists all items or recipes in the inventory
```
Comment (aqcd) [inline docs/UserGuide.md]: will this ever be supported?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -213,64 +205,68 @@ NOT Supported as of v1.2 yet
...
+### Delete item/recipe : `del`
 NOT Supported as of v1.2 yet
 
 **NAME:**
-- `del` - delete an item
+- `del` - deletes an item or recipe
 
 **SYNOPSIS:**
-- `del` \<item name\> \[-r index\]
+- `del` **-n \<item name\>** \[-r index\]
```
Comment (aqcd) [inline docs/UserGuide.md]: same for this

File: docs/UserGuide.md (hunk 1 above)
Comment (Rahul0506) [reply docs/UserGuide.md]: Done, my bad

**Review by aqcd — APPROVED** [review]
lgtm

**Pull request merged**